index.html

```diff
@@ -51,5 +51,201 @@ <h3 class="connect">LET'S CONNECT</h3>
         </section>
       </div>
     </section>
+    <section class="work">
+      <div class="card card-1">
+        <a href="#"
+          ><img
+            src="images/Snapshoot-Portfolio.png"
+            alt="Snapshoot-Portfolio"
+            class="snapshot"
+        /></a>
+        <p class="tonic flex0">Tonic</p>
```

Comment:
- [ ] Let's make this code more semantic instead of using  `<p class="tonic flex0">Tonic</p>`, try using an `hx` tag. Do this also for the rest of your cards, please.

This is important because structural, semantic HTML is the key starting point toward good accessibility practices. Many screen readers have functionality that allows a user to select to read only the headings on the page or only the links. Giving precedence to the way headings and links are written is a significant way we can make browsing the web easier for these users. :computer: :wink: